### ISSUE TITLE
Allow configuration of discovery interval

### ISSUE THREAD

**Issue by frittentheke** [hivemq/hivemq-dns-cluster-discovery-extension#4]
According to https://www.hivemq.com/docs/4.2/extensions/registries.html#extension-cluster-discovery the default interval for running the reload function of the ClusterDiscoveryCallback is 60 seconds. Currenty there is only a configurable timeout for DNS resolution, but no way to tune the interval. 

According to the SPI https://github.com/hivemq/hivemq-extension-sdk/blob/0936ee7161dedcef999c69efc9ffc84ac3a6df85/src/main/java/com/hivemq/extension/sdk/api/services/cluster/ClusterDiscoveryCallback.java#L53 this can quite easily be configured and would, especially in the case of DNS reduce the time it takes for the cluster to converge.

60 seconds with a potential TTL on queried A-records of i.e. 60 seconds (and a local DNS cache such as systemd-resolved) could result in a worse-case of 2 minutes for HiveMQ to recognize its fellow cluster members and only then to start joining / merging the cluster.

To cut a long story short: I'd like to suggest to allow for individual configuration of the interval the DNS discovery is run via its config file.

**Comment by frittentheke** [comment]
@sbaier1 your PR https://github.com/hivemq/hivemq-dns-cluster-discovery-extension/pull/5 appears to implement the suggested settings. What's left to in order to get it merged?

**Comment by SgtSilvio** [comment]
@frittentheke yes, this issue can now be closed. Sorry that your issue was a long time not on our radar.
